src/Sema.zig

```diff
@@ -31330,8 +31331,10 @@ fn intAddWithOverflowScalar(
 }
 
 /// Asserts the values are comparable. Both operands have type `ty`.
-/// Vector results will be reduced with AND.
-fn compare(
+/// For vectors, returns true if the comparison is true for ALL elements.
+///
+/// Not that `!compareAll(.eq, ...) != compareAll(.neq, ...)`
```

Comment:
Typo: "Not"

Reply:
Nice catch. Rather confusing the way I left it, hah